versioned_docs/version-2.0.0/running-keploy/rename-testcases.md

```diff
@@ -1,53 +1,51 @@
 ---
 id: rename-testcases
-title: Rename Default Test-names
-sidebar_label: Rename Testname
-description: This section documents how to rename testcases & testsets
+title: Rename TestCases
+sidebar_label: Rename Testcases
+description: This section documents how to rename testcases
 tags:
   - rename testcases
-  - rename testsets
 keywords:
   - testcases
   - documentation
   - rename
-  - testsets
 ---
 
-## Rename Test-Cases
+First Ensure keploy is running in record mode.
 
-To name your test case, make an API call and **add a header field** whose key is set to `Keploy-Test-Name` and value is set to the name you want for your test case.
-
-### Example
+To name your testcase, make an api call and add a header field whose key is set to `Keploy-Test-Name` and value is set to the name you want for your testcase.
 
 To demonstrate this, let's consider the [gin-mongo](https://github.com/keploy/samples-go/tree/main/gin-mongo) sample application.
 
-1. Clone the repository and build the application.
+1. Clone the repository and move to gin-mongo folder
 
 ```bash
 git clone https://github.com/keploy/samples-go && cd samples-go/gin-mongo
+```
+
+2. Create a binary for the application.
+
+```bash
 go build
 ```
 
-2. Run keploy in record mode and provide the path to the binary.
+3. Run keploy in record mode and provide the path to the binary.
 
 ```bash
 sudo -E keploy record -c "./test-app-url-shortener"
 ```
 
-3. Make an api call and add a header field with key set to `Keploy-Test-Name` and value set to the name you want for your testcase, let's say 'renametest'.
+4. Make an api call and add a header field with key set to `Keploy-Test-Name` and value set to the name you want for your testcase, let's say 'renametest'.
 
-```bash
+```shell
 curl --request POST \
   --url http://localhost:8080/url \
   --header 'content-type: application/json' \
-  --header 'Keploy-Test-Name: renamed-test' \
+  --header 'Keploy-Test-Name: renametest' \
   --data '{
   "url": "https://google.com"
 }'
 ```
 
-5. Test case with the name `renamed-test.yaml` has been successfully created. 🎉
-
-## Rename Test-Sets
-
-To rename your test set, you can manually override the default name from `test-set-0` to a `kTest-0` in the `keploy` folder.
```

Comment:
Any specifics to remove the guide to update the renaming of test-set folders step?
Also in the description itself is mentioned that this guide is how to rename both test-set and testcases, so any nudge on there?